widgetssdk/src/main/res/layout/app_bar.xml

```diff
@@ -21,17 +21,36 @@
             android:layout_gravity="center"
             android:textAppearance="?attr/textAppearanceHeadline2" />
 
-        <com.glia.widgets.view.header.button.GliaEndButton
-            android:id="@+id/end_button"
-            style="?attr/gliaHeaderEndButtonStyle"
+        <LinearLayout
+            android:id="@+id/button_layout"
+            android:orientation="horizontal"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:layout_gravity="end"
-            android:layout_marginEnd="@dimen/glia_large"
-            android:minWidth="@dimen/glia_app_bar_end_button_width"
-            android:text="@string/glia_top_app_bar_chat_end"
-            android:contentDescription="@string/glia_top_app_bar_chat_end_content_description"
-            android:visibility="gone" />
+            android:layout_marginEnd="@dimen/glia_large">
+
+            <androidx.appcompat.widget.AppCompatImageView
```

Comment:
I'd suggest using ImageButton in such cases to make it recognizable by accessibility tools as a Button instead of an Image. But particularly for this case, I think it's better to remove the whole LinearLayout and put the icon into the menu and make `android:visible="false"` by default

Reply:
Done